packages/ruby.rb

```diff
@@ -3,36 +3,32 @@
 class Ruby < Package
   description 'Ruby is a dynamic, open source programming language with a focus on simplicity and productivity.'
   homepage 'https://www.ruby-lang.org/en/'
-  version '3.0.0-1'
+  version '3.0.0-2'
   compatibility 'all'
   source_url 'https://cache.ruby-lang.org/pub/ruby/3.0/ruby-3.0.0.tar.gz'
   source_sha256 'a13ed141a1c18eb967aac1e33f4d6ad5f21be1ac543c344e0d6feeee54af8e28'
 
-  binary_url ({
-     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ruby-3.0.0-1-chromeos-armv7l.tar.xz',
-      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ruby-3.0.0-1-chromeos-armv7l.tar.xz',
-        i686: 'https://dl.bintray.com/chromebrew/chromebrew/ruby-3.0.0-1-chromeos-i686.tar.xz',
-      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ruby-3.0.0-1-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ruby-3.0.0-2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ruby-3.0.0-2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/ruby-3.0.0-2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ruby-3.0.0-2-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-     aarch64: '0849babd7e1abb994314af2a04b3612085e7ca19a91cbd50642cbccb092b577c',
-      armv7l: '0849babd7e1abb994314af2a04b3612085e7ca19a91cbd50642cbccb092b577c',
-        i686: 'ed9cfb4bd99c3423b2a464c07c748820ec3b640f4bc13d4c6c8b1d88b76f1e89',
-      x86_64: '0b21fef16a94cfb9e2815f028d2e8aad2c32554d09ff09d485a2fc5dcfa06c34',
+  binary_sha256({
+    aarch64: '6e2d40ea5d1bafbe0b79e1a0ae644911f33a9ceb94716d3e4c05d91884850b28',
+     armv7l: '6e2d40ea5d1bafbe0b79e1a0ae644911f33a9ceb94716d3e4c05d91884850b28',
+       i686: '44422c2fbe7c1d4d8d0088d12c1691d33de19fbe10a1461d9b3ce8cf1e2e7e1e',
+     x86_64: '20d4d18d1a184a088abb5547bc94e7f6189229565157bb35c5691cf0f1264f9f'
   })
 
-  depends_on 'readline'
-  depends_on 'ncurses'
-  depends_on 'zlibpkg' => :build
-  depends_on 'openssl' => :build
-  # depends_on 'libyaml' # This is needed to install gems
+  depends_on 'libyaml' # This is needed to install gems
```

Comment:
Missing `depends_on 'ca_certificates'`?

Reply:
Should we then add this to install.sh as well? Core packages?

Reply:
(and yes ca_certificates fixes the openssl issue.)

Reply:
Yes, if this is a dependency for ruby then it needs to be added to install.sh before ruby is executed.  `tools/core_packages.txt` will need to be updated as well.